README.md

```diff
@@ -4,6 +4,6 @@
 2. ```vagrant up```
 3. ```vagrant ssh```
 4. ```cd /project_data/```
```

Comment:
Serhiy, why don't we name our source folder on the virtual machine "ecomap" instead of "project_data" to be consistent?